### PR TITLE
Upgrade eslint

### PR DESCRIPTION
**Pull request by erwinmombay**
need to upgrade eslint to support `"object-shorthand": [2, "properties", { "avoidQuotes": true }]`

### CODE REVIEW

File: ads/google/a4a/test/test-utils.js [hunk 1]
```diff
@@ -148,15 +148,15 @@ describe('Google A4A utils', () => {
         setupForAdTesting(fixture);
         let url;
         const headers = {
-          get: function(name) {
+          get(name) {
```
Comment (erwinmombay) [inline ads/google/a4a/test/test-utils.js]: @cramforce do we prefer this? I think the eslint autofix redid all of these, i'll revert if these have perf implications

**Comment by cramforce** [comment]
I also saw what @rsimha-amp saw. Do we allow both now? If yes, this is certainly better than before. It seems to only fix completely off indents.

File: ads/google/a4a/test/test-utils.js (hunk 1 above)
Comment (cramforce) [inline ads/google/a4a/test/test-utils.js]: Does this not compile to the same code?

Comment (erwinmombay) [inline ads/google/a4a/test/test-utils.js]: i double checked CC and it does. do we prefer it though?

cc @jridgewell

**Comment by erwinmombay** [comment]
i would prefer 4 indent as well since that is google style guide, im trying to figure out if its possible or not since this was just autofixed by eslint

File: ads/google/a4a/test/test-utils.js (hunk 1 above)
Comment (jridgewell) [inline ads/google/a4a/test/test-utils.js]: They mean the same.

File: ads/caajainfeed.js [hunk 2]
```diff
@@ -23,30 +23,30 @@ import {loadScript, validateData} from '../3p/3p';
 export function caajainfeed(global, data) {
 
   validateData(
-    data,
+      data,
     [],
```
Comment (rsimha-amp) [inline ads/caajainfeed.js]: Shouldn't this line have gotten indented too?

File: ads/google/a4a/line-delimited-response-handler.js [hunk 3]
```diff
@@ -72,9 +72,9 @@
    return function(line, done) {
      if (first) {
        callback(
-         unescapeLineDelimitedHtml_(line),
+           unescapeLineDelimitedHtml_(line),
          /** @type {!Object<string, *>} */(tryParseJson(first) || {}),
```
Comment (rsimha-amp) [inline ads/google/a4a/line-delimited-response-handler.js]: Same here. Looks like this should've been indented 4 spaces.

Comment (erwinmombay) [inline ads/google/a4a/line-delimited-response-handler.js]: yeah seems like a bunch of eslint autofix bugs

**Pull request merged**